ansible/roles/dokuwiki/templates/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "2"
+
+services:
+
+  dokuwiki:
+    image: istepanov/dokuwiki:2.0
+    volumes:
+      - dokuwiki:/var/dokuwiki-storage
+    ports:
+      - 61906:80
+
+volumes:
+  dokuwiki:
```

Comment:
This makes me feel weird. Is there no other way of writing this without a trailing `:`?